cypress/e2e/productManager.cy.js

```diff
@@ -0,0 +1,125 @@
+/// <reference types="cypress" />
+
+describe('상품관리 페이지 테스트', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:5000/index.html');
+    cy.get('#product-manage-menu').click();
+  });
+
+  it('최초 상품 목록은 비워진 상태이다.', () => {
+    cy.get('#product-name-input').should('be.empty');
+    cy.get('#product-price-input').should('be.empty');
+    cy.get('#product-quantity-input').should('be.empty');
+
+    cy.get('#product-inventory-container').children().should('have.length', 0);
+  });
+
+  it('상품 이름은 공백이 불가능하다.', () => {
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('상품 이름을 입력해주세요!');
+    });
+
+    cy.get('#product-add-button').click();
+  });
+
+  it('상품 금액은 공백이 불가능하다.', () => {
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('상품 가격을 양수로 입력해주세요!');
+    });
+
+    cy.get('#product-name-input').type('name');
+    cy.get('#product-add-button').click();
+  });
+
+  it('상품 금액의 최소 가격은 100원', () => {
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('상품 가격을 양수로 입력해주세요!');
+    });
+
+    cy.get('#product-name-input').type('name');
+    cy.get('#product-price-input').type(5);
```

Comment:
경계값을 테스트하는게 좋다고 생각해요

```suggestion
    const PRODUCT_MIN_PRICE = 100;
    cy.get('#product-price-input').type(PRODUCT_MIN_PRICE - 1);
```